influxdb_iox_client/src/client/management.rs

```diff
@@ -644,4 +668,28 @@ impl Client {
             .operation
             .ok_or(ClosePartitionChunkError::EmptyResponse)?)
     }
+
+    /// Wipe preserved catalog of specified, but non-existing database.
```

Comment:
```suggestion
    /// Clear all potential preserved catalog files for the specified database, regardless of
    /// current state. 
```

I was a little confused about "non-existing" -- is this accurate?

Reply:
I've changed it to:

```text
Wipe potential preserved catalog of an uninitialized database.
```

because "regardless of the current state" is not true, since you cannot wipe an initialized / online database. We might wanna add this feature at some point, but right now "wipe" is purely meant to be a recovery mechanism.